README.md

```diff
@@ -1,3 +1,270 @@
 # DDM React CropperJS
 
-# TODO add example of using the component and document the props
+[CropperJS](https://github.com/fengyuanchen/cropperjs) wrapped in a React component
+
+See the (CropperJS Documentation)[https://github.com/fengyuanchen/cropperjs#usage] for more information
```

Comment:
Not a huge deal but it should be [CropperJS Documentation](link) instead of (CropperJS Document)[link]